### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by stephenhebert** [shentao/vue-multiselect#1485]
IIUC, this library / component doesn't seem to support accessibility for visually-impairment / screen-readers.  

I'm noticing things like missing labels on the options in the dropdown list and missing labels (or id attribute) on the input when the `searchable` option is `true`. 

To reproduce, just use any of the demos on the vue-multiselect homepage with a screen reader. (I'm currently using ChromeVox.)

Is there currently any plan to add support for accessibility?

**Comment by vincerubinetti** [comment]
I found the same problem with `@vueform/multiselect`, an oft-recommended alternative to this library. If you look at the HTML for this component, you see no `aria-`/`role`/etc attributes. Just divs with tabindex. That's not semantic enough info for a screenreader to make sense of.

[`vue-select`](https://github.com/sagalbot/vue-select) seems to be a possible alternative that considers accessibility. Not sure why I hadn't seen it before. Maybe I thought it was only single select, but it has multiple.

**Comment by laurens94** [comment]
Related: https://github.com/shentao/vue-multiselect/issues/927
